tensorboard/util.py

```diff
@@ -0,0 +1,224 @@
+# Copyright 2017 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""TensorBoard helper routine module.
```

Comment:
I'm actually planning to move this to `tf_logging.py` in the TensorFlow codebase, so all TensorFlow users can benefit from awesome logs.

The way I typically use util is it's just a place where I toss stuff, used in >1 places, when I don't want to create an independent package/module and build rule. The goal is to not have the util package pull in any heavyweight dependencies that cause everything to be dragged down.